webapp/tests/forms/flows/test_lotse_flow.py

```diff
@@ -1171,6 +1171,15 @@ def test_if_handwerker_filled_then_set_next_url_correct(self):
                 self.haushaltsnahe_step, self.render_info_haushaltsnahe_step, {'familienstand': 'single'})
             self.assertEqual(self.haushaltsnahe_yes_url, render_info.next_url)
 
+        # separated
+        with self.app.test_request_context(method='POST', data=self.data_haushaltsnahe_yes):
+            render_info, _ = self.flow._handle_specifics_for_step(
+                self.haushaltsnahe_step, self.render_info_haushaltsnahe_step,
+                {'familienstand': 'married', 'familienstand_married_lived_separated': 'yes',
+                 'familienstand_married_lived_separated_since': datetime.date(1990, 1, 1)})
+            self.assertEqual(self.haushaltsnahe_yes_url, render_info.next_url)
```

Comment:
What is with all the other cases (widowed etc.)?

Additionally, I do not think that we need that part here because we test that already below, don't we?

Reply:
Mh, no, here we test the separated case, below the person is only married. But sure, I can add the other cases.